src/supertux/menu/options_menu.cpp

```diff
@@ -235,6 +236,10 @@ OptionsMenu::OptionsMenu(bool complete) :
     add_toggle(MNID_CHRISTMAS_MODE, _("Christmas Mode"), &g_config->christmas_mode);
   }
 
+  add_toggle(MNID_CONFIRMATION_DIALOG, _("Confirmation Dialog"), &g_config->confirmation_dialog)
+    ->set_help("Confirm aborting level");
+  std::cerr << g_config->confirmation_dialog << std::endl;
```

Comment:
Use log_debug instead of cerr or delete (this does not seem to be needed anymore).